locksmith/src/routes/price.ts

```diff
@@ -4,5 +4,6 @@ const router = express.Router()
 const priceController = require('../controllers/priceController')
 
 router.get('/:lockAddress', priceController.price)
+router.get('/fiat/:lockAddress', priceController.fiatPrice)
```

Comment:
We could probably include the fiat price as at attribute under the original call, not sure if there is a  technical reason but this could make this a smaller change in  PR and a smaller change for callers to switch to

Reply:
So the original call does not have the logic of not returning the price for locks which are not approved which I think is fine because we had other requirements. 
Here, I am trying to think forward of the approach we will eventually take where the front end will not care about what kind of lock that is, what kind of purchase (pruchaseFor or grantKey) that is... etc.

The front end will just display things and pass things to locksmith.

Reply:
Also I do not want to "break" things/flows which are working